wgsl/index.bs

```diff
@@ -520,6 +520,76 @@ type.
 
 ## Texture Types TODO ## {#texture-types}
 
+A <dfn noexport>texel</dfn> is a scalar or vector used as the smallest independently addressible element of a [=texture=].
+The word *texel* is short for *texture element*.
+
+A <dfn noexport>texture</dfn> is a grid of texels supporting special operations useful for rendering.
```

Comment:
nit: multi-dimensional grid (it can be 1, 2, 3 or even 4 if when we support 2D array multisampled textures)

Reply:
The bullet points below describe dimensionality, and say it's 1,2,3 dimensions. There is also array-size, when the texture is arrayed.

Note that the compute shader workgroup-grid is 3-dimensional without explicitly calling it out as multi-dimensional.